### PR TITLE
JBPM-5990 - New date range picker

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Comment by cristianonicolai** [comment]
Depends on https://github.com/AppFormer/uberfire/pull/756

**Comment by cristianonicolai** [comment]
@nmirasch @bsremac @rbarriuso please take a look when you have a chance.

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceSummaryErrorPopoverCell.java [hunk 1]
```diff
@@ -45,84 +43,88 @@
 
     @Inject
     @DataField("popover")
-    private Popover popover;
```
Comment (rbarriuso) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceSummaryErrorPopoverCell.java]: @cristianonicolai I'm curious: why finally not using the `Popover` widget?

Comment (rbarriuso) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceSummaryErrorPopoverCell.java]: I understand the reason is https://github.com/AppFormer/uberfire/pull/756 provides a more flexible way of using Patternfly popovers on different elements, please correct me if I'm wrong

Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceSummaryErrorPopoverCell.java]: @rbarriuso exactly, the previous approach only really worked for `Anchor` elements, but if you look at the JavaScript API that isn't true, basically any element could have a popover.

**Pull request merged**